packages/shared/flows.yaml

```diff
@@ -2877,6 +2877,101 @@ integrations:
                         links:
```

Comment:
This file will soon become unmanageable :D

Reply:
Yeah, I've never touched this by hand, `flows.js` just updates this programmatically.